Islandora-Devops/islandora-starter-site
# update citation select to 2.x to remove bibcite dependency

Update citation_select to [2.0.0-beta1](https://www.drupal.org/project/citation_select/releases/2.0.0-beta1) release.

* Removes bibcite dependency (brings in the necessary style sheets and config to this module)
* Uses seboettg/citeproc-php to generate the citations
* Additional logic to handle more cases of EDTF dates, including ranges
* Picks the first publisher when there are more than one

How to test it:
* Get the PR
* Create some sample content
* Verify that citations are as expected
* Verify bibcite module is no longer installed
